spec/rails_helper.rb

```diff
@@ -1,25 +1,30 @@
 # frozen_string_literal: true
 
 # This file is copied to spec/ when you run 'rails generate rspec:install'
-ENV['RAILS_ENV'] ||= "test"
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
Still seeing double quote to single quote changes. Can we revert these and turn off this cop?